sage.py

```diff
@@ -128,11 +165,12 @@ def _get_data(self):
         logging.debug("Retrieving scans and scan summaries (aka scan history)")
         self.data['scans'] = self.hub.get_codelocations(limit=99999).get('items', [])
         for scan in self.data['scans']:
-            try:
-                scan_summaries = self.hub.get_codelocation_scan_summaries(code_location_obj = scan).get('items', [])
-            except:
```

Comment:
I took this try/except out as it was masking the problem. We should not hit the JSONDecodeError now and if we do, there is something else more seriously wrong.

Reply:
Makes sense. At some point we will enhance the blackduck library to handle this more elegantly, but for now this work around makes sense. Thanks.